pkg/apis/troubleshoot/v1beta2/supportbundle_interfaces.go

```diff
@@ -0,0 +1,58 @@
+package v1beta2
+
+// the intention with these appends is to swap them out at a later date with more specific handlers for merging the spec fields
+func (s *SupportBundle) ConcatSpec(bundle *SupportBundle) {
+
+    s.Spec.MergeCollectors(&bundle.Spec)
+
+	for _, v := range bundle.Spec.AfterCollection {
+		s.Spec.AfterCollection = append(s.Spec.AfterCollection, v)
+	}
+	for _, v := range bundle.Spec.HostCollectors {
+		s.Spec.HostCollectors = append(s.Spec.HostCollectors, v)
+	}
+	for _, v := range bundle.Spec.HostAnalyzers {
+		s.Spec.HostAnalyzers = append(s.Spec.HostAnalyzers, v)
+	}
+	for _, v := range bundle.Spec.Analyzers {
+		s.Spec.Analyzers = append(s.Spec.Analyzers, v)
+	}
+}
+
+func (s *SupportBundleSpec) MergeCollectors(spec *SupportBundleSpec) {
+    for _,c := range spec.Collectors {
+        // we want to move away from checking for specific collectors in favor of allowing collectors to expose their own merge method
+        // ideally we'd just want to be able to call something like c.Merge(spec) here and have the collector's own merge method work out what to do'
+
+        if c.ClusterInfo != nil {
+            // we only actually want one of these so skip if there's already one
+            y := 0
```

Comment:
this should be a boolean with a meaningful name, like `hasClusterInfo := false`

Reply:
I removed these in favor of addressing them properly in another task